00-programming-fundamentals/intro-to-automated-tests.md

```diff
@@ -124,91 +124,108 @@ We are now setup, but we haven't actually written a test yet.
 
 ```ruby
-class TestDie < MiniTest::Unit::TestCase
+# die_spec.rb
+require 'minitest/autorun'
+require 'minitest/reporters'
+require_relative 'die'
+
+describe "die" do
 
 end
 ```
 
-The `< MiniTest::Unit::TestCase` indicates that this class gets all the methods & instance variables of the TestCase class in Minitest.  
+The `describe` block indicates that this contains a suite of tests.  Minitest will run these tests together and any instance variables created inside this block will be shared among the test cases.  
 
-#### Step 2:  Create a `test_creation_of_die` method.
+#### Step 2:  Create an `it` block
 
-Now we'll create a method called `test_creation_of_die`, this is a test-case.  All test-cases in assert-style Minitest must start with "test_".  
+Now we'll create an `it` block which is a test-case.  Each `describe` block can have many `it` blocks inside it and even many `describe` blocks.  Each `it` block however should focus on testing one specific thing, while `describe` blocks are used to group a set of tests.
 
 
 ```ruby
-# die_test.rb
+# die_spec.rb
 
 require 'minitest/autorun'
 require 'minitest/reporters'
 require_relative 'die'
 
-class TestDie < MiniTest::Unit::TestCase
-  def test_creation_of_die
-
+describe "die" do
+  it "Can be created" do
+    # Testing goes in here
   end
 end
 ```
 
-#### Step 3:  Add an assertion
-
+#### Step 3:  Add an expectation
 
-So we have a test-case, but it's not actually checking anything yet.  So we can add an _assertion_ which is a method that **asserts** that a specific condition must be true.  In this case we need to **assert** that if we create a die, it's an instance of the Die class.
 
-So we create a 
+So we have a test-case, but it's not actually checking anything yet.  So we can add an _expectation_ which is a method call that describes a condition it **expects** the given item to meet.  In this case we need to **expect** that if we create a die, it's an instance of the Die class.  Expectations typically start with `must_` or `wont_`.
 
+So lets write our expectation
 
 ```ruby
-# die_test.rb
+# die_spec.rb
 
 require 'minitest/autorun'
 require 'minitest/reporters'
 require_relative 'die'
 
-class TestDie < MiniTest::Unit::TestCase
-  def test_creation_of_die
+describe "die" do
+  it "Can be created" do
     @die = Die.new
-    # The class of @die should be Die
-    assert @die.class == Die, "There must be a Die class."
+    # the class of @die should be Die
+    @die.class.must_equal Die
   end
 end
 ```
 
-Notice the line `assert @die.class, Die`.  The `assert` method takes two required arguments and an optional message.  If the two required arguments are equal, the test passes, otherwise it fails.  
+Notice the line `@die.class.must_be_instance_of Die`.  Minitest adds the `must_equal` method to all objects and it takes one required argument.  If the required argument is equal to the object `must_equal` is being called on, the expectation passes, otherwise it fails.  
```

Comment:
Here you say `must_be_instance_of Die`, but above you say `.class.must_equal Die`.